pdata/metadata.yaml

```
@@ -1,8 +1,11 @@
 type: pdata
+github_project: open-telemetry/opentelemetry-collector
 
 status:
   class: pkg
   codeowners:
     active:
       - BogdanDrutu
       - dmitryax
+  stability:
+    stable: [traces, metrics, logs]
```

```suggestion
    stable: [traces, metrics, logs]
    development: [profiles]
```

not sure if profiles is in development or alpha, but would be good to include

---

Pressed the merge button to fast. Profile is not in this package but in the xprofile.